blog/archives/2018-10-25.json

```diff
@@ -2,8 +2,8 @@
   "location": "Code & Cafe, 4304 Rue Saint-Denis, Montreal, QC",
   "gmapIframe": "https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2795.3883541630685!2d-73.57987!3d45.522389999999994!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0x0!2zNDXCsDMxJzIwLjYiTiA3M8KwMzQnNDcuNSJX!5e0!3m2!1sfr!2sca!4v1566585784205!5m2!1sfr!2sca",
   "backgroundImg": "https://secure.meetupstatic.com/photos/event/6/2/b/3/highres_475645267.jpeg",
-  "hour": "18h30 - 21h00",
-  "date": "October 25th, 2018",
+  "startAt": "2018-10-25T18:30:00-05:00",
```

Comment:
Moving to ISO dates to improve date manipulation and help for localisation in the future.

Reply:
i like the idea of using a data but as i want it really simple it could be better if we can use something like dd/mm/yyyy and hours and from here convert to a date. It can be "hard" to simple person to create a Date() and being sure to use the good timezone

Reply:
Just saw this! I also really prefer the standard ISO approach. 
I'm not too worried about making it hard for future contributors because: 
1) ISO is a best practice of the industry when dealing with dates
2) Anyone wanting to add a new event will just copy paste the previous event and simply change the numbers in place - low chances of mistake here
3) If someone does not understand this 1) it will be a change to further their knowledge towards ISO dates, and grow as a dev 2) if they actually screw it up, it won't work at display and 3) worst of the worst case, they do a PR with it and we will gladly let them know the why and the how of ISO dates 
4) Stephane already did the required change in the template to make it work :D

Reply:
Also, just thought of one more and actually super important point: 
5. it will allow for a super easy translation of strings right in the template when we get to implementing i18n on the site -- as opposed to duplicate dates to have them in En/Fr in the json file